sync/handlers/handler.go

```diff
@@ -35,6 +37,7 @@ type syncHandler struct {
 	stateTrieLeafsRequestHandler *LeafsRequestHandler
 	blockRequestHandler          *BlockRequestHandler
 	codeRequestHandler           *CodeRequestHandler
+	signatureRequestHandler      handlers.SignatureRequestHandler
```

Comment:
We should move the handler out of `sync/handlers/handler.go` not only the `SignatureRequestHandler`. Sync refers to state sync and this is now a more general network request handler.